gensim/models/ldamodel.py

```diff
@@ -579,6 +579,15 @@ def update(self, corpus, chunksize=None, decay=None, offset=None,
         if gamma_threshold is None:
             gamma_threshold = self.gamma_threshold
 
+        if isinstance(corpus, list):
+            logger.warning(
+                "Taking plain list as a corpus for LDA.\n"
+                "We strongly recommend you to use corpus streaming instead of lists. "
+                "LDA is processing the corpus in batches and "
+                "it only loads one batch of documents into memory at once.\n"
```

Comment:
-1 on newlines in log lines. Makes the log harder to process and parse (missing timestamps, broken format).

If you must output multiple lines, output them individually.